Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -6,7 +6,7 @@
 class Keg
   def relocate_dynamic_linkage(relocation)
     # Patching the dynamic linker of glibc breaks it.
-    return if name == "glibc"
+    return if name == "glibc" || name.start_with?("glibc@")
```

Comment:
```suggestion
    return if name.match? Version.formula_optionally_versioned_regex(:glibc)
```
`Version` should be available here, right?